srb2kart.sh

```diff
@@ -5,7 +5,14 @@ for i in {0..9}; do
 done
 
 export PATH="/app/utils/gamescope/bin:$PATH"
-export SRB2WADDIR=/app/extra
+
+if [[ -z "$SRB2WADDIR" ]]; then
+	export SRB2WADDIR=/app/extra
+fi
+
+if [[ -z $(find ~/.srb2kart -name bonuschars.kart) ]]; then
+	ln -s /app/extra/bonuschars.kart ~/.srb2kart/bonuschars.kart
+fi
```

Comment:
Wouldn't we want to override this link each time the app is run regardless of whether it was previously set?

Reply:
i was thinking that someone would like to organize their mods, so if you dont want it to be somewhere it would be annoying for it to just appear there again for what would appears to be no reason.